compiler/utils/pos.ml

```diff
@@ -125,24 +133,26 @@ let retrieve_loc_text (pos : t) : string =
           "\n"
           ^
           if line_no = sline && line_no = eline then
-            Cli.with_style error_indicator_style "%*s"
-              (get_end_column pos - 1)
-              (String.make
+            Cli.with_style error_indicator_style "%*s%s"
+              (get_start_column pos - 1)
+              ""
+              (string_repeat
                  (max (get_end_column pos - get_start_column pos) 0)
-                 '^')
+                 "‾")
           else if line_no = sline && line_no <> eline then
-            Cli.with_style error_indicator_style "%*s"
-              (String.length line - 1)
-              (String.make
+            Cli.with_style error_indicator_style "%*s%s"
+              (get_start_column pos - 1)
+              ""
+              (string_repeat
                  (max (String.length line - get_start_column pos) 0)
-                 '^')
+                 "‾")
```

Comment:
Ah! I didn't know this style of error messages. What compiler does it try to mimic?

Reply:
Nothing specific, but this is relatively standard

This is as much as I could find for standard: https://github.com/typester/emacs/blob/master/etc/compilation.txt#L203

So maybe a better choice could be a `.` instead of the `.`:
- single line `file:line.col-col:`
- multiple lines `file:line.col-line.col:

this might give us a better chance for editors to automatically highlight them

Reply:
Ah no I was referring to the `‾` instead of `^`

Reply:
Ah sorry! It doesn't mimic anything specific, it's just an unicode underline :shrug: